project/src/main/java/com/google/sps/Models/Document.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.models;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class Document{
+  String language, name, hash;
+  ArrayList<Long> userIDs = new ArrayList<Long>();
+  
+  Document(String name, String language, String hash, ArrayList<Long> userIDs) {
```

Comment:
Why is userIDs an arraylist? I thought this would be a single user who owns the document - do we expect one document have multiple owners?

Reply:
I am guessing these are the ppl who can access the doc? Who will be the owner?
Comments will help clarify these things.

Reply:
// userIDs is a list of people who can access the current document
// permissions are not setup yet so "owners" and "collaborators" have not yet been setup

I'll add this comment!

Reply:
Is this comment added in this PR or in a separate one? I don't see it in the diff - prefer to add it in the same PR if possible.